Makefile

```diff
@@ -171,51 +176,40 @@ dockerpush: dockerbuild
 
 
 .PHONY: dockerrun
-dockerrun: dockerbuild
-	echo "Starting docker container and mapped its 8080 port to $(HTTP_PORT); http://localhost:$(HTTP_PORT)"
-	LOGS_DIR=/app/logs docker run -it -v $(LOGS_DIR):/app/logs -p $(HTTP_PORT):8080 $(DOCKER_IMG_LOCAL_TAG)
-
-
-# Spec targets
+dockerrun: clean_logs dockerbuild $(LOGS_DIR)
+	docker run \
+		-it -p $(HTTP_PORT):8080 \
+		--env-file=${PWD}/${ENV_FILE} \
+		--env LOGS_DIR=/logs \
+		--mount type=bind,source="${LOGS_DIR}",target=/logs \
+		$(DOCKER_IMG_LOCAL_TAG)
 
-.PHONY: lint-spec
-lint-spec:
-	docker run --volume "$(PWD)":/data jamescooke/openapi-validator -e openapi.yml
 
+ # Clean targets
 
-.PHONY: serve-spec
-serve-spec-redoc:
-	docker run -it --rm -p 8080:80 \
-		-v "$(PWD)/openapi.yml":/usr/share/nginx/html/openapi.yml \
-		-e SPEC_URL=openapi.yml redocly/redoc
-
-
-.PHONY: serve-spec-swagger
-serve-spec-swagger:
-	echo "SWAGGER UI on http://localhost:8080/swagger"
-	docker run -p 8080:8080 \
-		-e BASE_URL=/swagger -e SWAGGER_JSON=/openapi.yaml \
-		-v ${PWD}/openapi.yml:/openapi.yaml \
-		swaggerapi/swagger-ui
-
-
-# Clean targets
+.PHONY: clean_logs
+clean_logs:
+	rm -rf $(LOGS_DIR)
 
 .PHONY: clean_venv
 clean_venv:
 	pipenv --rm
 
 
 .PHONY: clean
-clean: clean_venv
+clean: clean_venv clean_logs
 	@# clean python cache files
 	find . -name __pycache__ -type d -print0 | xargs -I {} -0 rm -rf "{}"
+	rm -rf $(PYTHON_LOCAL_DIR)
```

Comment:
PYTHON_LOCAL_DIR is probably not used anymore and could be removed from the makefile